book/src/drivers.md

```diff
@@ -66,7 +66,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [MooseFS](https://github.com/moosefs/moosefs-csi) | `com.tuxera.csi.moosefs` | v1.0 | A Container Storage Interface (CSI) Driver for [MooseFS](https://moosefs.com/) clusters. | Persistent | Read/Write Multiple Pods | Yes | 
 [NetApp](https://github.com/NetApp/trident) | `csi.trident.netapp.io` | v1.0, v1.1, v1.2 | A Container Storage Interface (CSI) Driver for NetApp's [Trident](https://netapp-trident.readthedocs.io/) container storage orchestrator | Persistent | Read/Write Multiple Pods | Yes | Raw Block, Snapshot, Expansion, Cloning
 [NexentaStor](https://github.com/Nexenta/nexentastor-csi-driver) | `nexentastor-csi-driver.nexenta.com` | v1.0, v1.1, v1.2 | A Container Storage Interface (CSI) Driver for NexentaStor | Persistent | Read/Write Multiple Pods | Yes | Snapshot, Expansion, Cloning, Topology
-[Nutanix](https://portal.nutanix.com/#/page/docs/details?targetId=CSI-Volume-Driver:CSI-Volume-Driver) | `"com.nutanix.csi"` | v0.3, v1.0 | A Container Storage Interface (CSI) Driver for Nutanix | Persistent | "Read/Write Single Pod" with Nutanix Volumes and "Read/Write Multiple Pods" with Nutanix Files | Yes | 
+[Nutanix](https://portal.nutanix.com/#/page/docs/details?targetId=CSI-Volume-Driver-v2_0%3ACSI-Volume-Driver-v2_0) | `com.nutanix.csi` | v0.3, v1.0, v1.2 | A Container Storage Interface (CSI) Driver for Nutanix | Persistent | "Read/Write Single Pod" with Nutanix Volumes and "Read/Write Multiple Pods" with Nutanix Files | Yes | Raw Block, Snapshot, Expansion, Cloning
```

Comment:
With the old link https://portal.nutanix.com/page/documents/details?targetId=CSI-Volume-Driver%3ACSI-Volume-Driver, I can actually view it and see what is supported.
With the new link https://portal.nutanix.com/#/page/docs/details?targetId=CSI-Volume-Driver-v2_0%3ACSI-Volume-Driver-v2_0, however, I only see a support page which asks for login credentials.
I think the old link is better.

Reply:
Good point, I updated the link to point to public github page. With this I do not have to update the link in future.